readme.md

```diff
@@ -34,25 +34,47 @@ If you have problems building any of the projects, take a look at the online [FA
 Introduction
 ==
 
-### What's New in 5.3.1
-
-- This patch release contains a fix for a recently discovered bug affecting apps consuming Mobile SDK 5.3.0 through cocoapods. These apps were unable to open a SmartStore created with Mobile SDK 5.2.0 or earlier.
-
-### What's New in 5.3
+### What's New in 6.0
 
 **OS Version Support**
-- iOS 11.0 is fully supported in Mobile SDK 5.3.
+- iOS 11 is fully supported in Mobile SDK 6.0.
+- The minimum iOS version we support has been bumped up from iOS 9 to iOS 10.
+
+**IDE Support**
+- Xcode 9 is the minimum version of Xcode required by Mobile SDK 6.0.
 
 **Library Upgrades**
-- We've updated SQLCipher to version 3.4.1.
+- We've updated React Native to version 0.50.4.
+- We've updated Cordova to version 4.5.4.
+
+**Login Enhancements**
+- In version 6.0, Mobile SDK enhances its authentication handling by adding identity provider services.
+- Identity providers help known users avoid reentering their Salesforce credentials every time they log in to a Mobile SDK app.
+- A Mobile SDK app can be used to provide authentication services to other Mobile SDK apps.
+- We have created a template that can be used with `forcedroid` that demonstrates this functionality. This template can be found [here](https://github.com/forcedotcom/SalesforceMobileSDK-AuthTemplates).
+
+**SmartStore Enhancements**
+- Mobile SDK 6.0 introduces the ability to define your SmartStore schemas through configuration files rather than code.
+- To define soups for the default global store, provide a file named `globalstore.json`.
+- To define soups for the default user store, provide a file named `userstore.json`.
+
+**SmartSync Enhancements**
+- Beginning in Mobile SDK 6.0, you can define sync configuration files and assign names to sync configurations.
+- You can use sync names to run, edit, or delete a saved sync operation.
+- You can define “sync down” and “sync up” operations through configuration files rather than code.
+- To define sync operations for the default global store, provide a file named `globalsyncs.json`.
+- To define sync operations for the default user store, provide a file named `usersyncs.json`.
+
+**Other Enhancements**
+- Mobile SDK 6.0 adds support for a developer settings screen in DEBUG builds.
```

Comment:
New in Mobile SDK 6.0, we are adding dev support screens. You do not need to write any code to use them. To access the dev support dialog, shake your device or select "Shake Gesture" inside the Hardware menu in the iOS Simulator. You can also use the ^⌘z keyboard shortcut when your app is running in the iOS Simulator.